user-storage-simple/src/main/java/org/keycloak/quickstart/writeable/PropertyFileUserStorageProvider.java

```diff
@@ -224,46 +202,47 @@ public UserModel addUser(RealmModel realm, String username) {
     @Override
     public boolean removeUser(RealmModel realm, UserModel user) {
         synchronized (properties) {
-            if (properties.remove(user.getUsername()) == null) return false;
+            if (properties.remove(user.getUsername()) == null)
+                return false;
             save();
             return true;
         }
     }
 
-
-
-
-
     // CredentialInputValidator methods
 
     @Override
     public boolean isConfiguredFor(RealmModel realm, UserModel user, String credentialType) {
         String password = properties.getProperty(user.getUsername());
-        return credentialType.equals(CredentialModel.PASSWORD) && password != null;
+        return credentialType.equals(PasswordCredentialModel.TYPE) && password != null;
     }
 
     @Override
     public boolean supportsCredentialType(String credentialType) {
-        return credentialType.equals(CredentialModel.PASSWORD);
+        return credentialType.equals(PasswordCredentialModel.TYPE);
     }
 
     @Override
     public boolean isValid(RealmModel realm, UserModel user, CredentialInput input) {
-        if (!supportsCredentialType(input.getType()) || !(input instanceof UserCredentialModel)) return false;
+        if (!supportsCredentialType(input.getType()) || !(input instanceof UserCredentialModel))
+            return false;
 
-        UserCredentialModel cred = (UserCredentialModel)input;
+        UserCredentialModel cred = (UserCredentialModel) input;
         String password = properties.getProperty(user.getUsername());
-        if (password == null || UNSET_PASSWORD.equals(password)) return false;
+        if (password == null || UNSET_PASSWORD.equals(password))
+            return false;
         return password.equals(cred.getValue());
     }
 
     // CredentialInputUpdater methods
 
     @Override
     public boolean updateCredential(RealmModel realm, UserModel user, CredentialInput input) {
-        if (!(input instanceof UserCredentialModel)) return false;
-        if (!input.getType().equals(CredentialModel.PASSWORD)) return false;
-        UserCredentialModel cred = (UserCredentialModel)input;
+        if (!(input instanceof UserCredentialModel))
+            return false;
+        if (!input.getType().equals(PasswordCredentialModel.TYPE))
+            return false;
+        UserCredentialModel cred = (UserCredentialModel) input;
```

Comment:
please revert formatting changes (and keep `CredentialModel.PASSWORD` -> `PasswordCredentialModel.TYPE`)

Reply:
Hi @vramik, one question, `CredentialModel.PASSWORD` is deprecated, should I keep it?